src/mame/drivers/gfamily.cpp

```diff
@@ -0,0 +1,120 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/*
+Games Family P4-4P (Pentium 4 - 4 Players)
+
+PC with Chinese Windows 2000 Pro and several emulators, including:
+ -MAME v0.78 (Dec 30 2003)
+ -MAME v0.96u3 (May 25 2005)
+ -MAME v0.106 (May 16 2006)
+ -Nebula 2.1.5
+ -FB Alpha v0.2.94.98
+ -ZiNc 1.1
+
+
+PC motherboard plus an additional PCB for JAMMA, inputs and basic config.
+
+PC board:
+CPU: Intel Celeron 1,7GHz / 128kb / 400MHz SL6SC
+RAM: 256MB-DDR400
+PCB: 04/082006-SiS-651-6A6IXRMAC-00, Realtec ALC655 audio, Realtec RTL8201BL Ethernet (25.000 MHz xtal), Winbond W83194BG-648 (14.31818 xtal)
```

Comment:
One of the frequencies here doesn’t have a unit.

Reply:
Done, thanks!